src/main/java/com/salesforce/mirus/config/SourceConfigDefinition.java

```diff
@@ -111,6 +112,13 @@ public static ConfigDef configDef() {
     for (SourceConfigDefinition f : SourceConfigDefinition.values()) {
       configDef = configDef.define(f.key, f.type, f.defaultValue, f.importance, f.doc);
     }
+
+    // Share name and all transform config definitions from ConnectorConfig
+    for (ConfigKey key : ConnectorConfig.configDef().configKeys().values()) {
+      if ("Transforms".equals(key.group) || ConnectorConfig.NAME_CONFIG.equals(key.name)) {
```

Comment:
Is this really case sensitive?

Reply:
The source is a protected constant inside Kafka Connect's [ConnectorConfig](https://github.com/apache/kafka/blob/a2e87feb8b1db8200ca3a34aa72b0802e8f61096/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java#L58), which is indeed capitalized. I could switch to `equalsIgnoreCase()` if we're worried about capitalization changing